resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -2363,7 +2363,7 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
             <stringProp name="filename"></stringProp>
```

Comment:
Track Service Request (Auth Lock Unlock) API - why is this not just "Auth Lock Unlock API "

Reply:
Done.